pkg/services/apiserver/auth/authorizer/org_id.go

```diff
@@ -60,6 +60,11 @@ func (auth orgIDAuthorizer) Authorize(ctx context.Context, a authorizer.Attribut
 		return authorizer.DecisionNoOpinion, "", nil
 	}
 
+	// If we have an access policy, let the API decide.
+	if signedInUser.GetIdentityType() == claims.TypeAccessPolicy {
+		return authorizer.DecisionNoOpinion, "", nil
```

Comment:
🤔 safer would be check that the namespace matches here -- but if that fails, I guess this is OK

Reply:
I don't think we have any native way to tie the provisioning user to the namespace per se, and not sure we want to require that in all future clients, either.

What is your concern with just checking identity type here?

Reply:
i dug more... and what i thought was possible is not... so i think OK